.rhdh/docker/Dockerfile

```diff
@@ -136,6 +136,14 @@ RUN \
   -e "s/(\"Last Commit\": \"(.+)\")/\1, \"Build Time\": \"$now\"/" && \
   cat packages/app/src/build-metadata.json; echo
 
+# Extract metadata values using grep/sed (no jq dependency) and create environment script
+RUN RHDH_VERSION=$(grep '"RHDH Version"' packages/app/src/build-metadata.json | sed 's/.*": "\([^"]*\)".*/\1/') && \
```

Comment:
should use jq here since it's json text.

Reply:
I tried using jq first but it was failing in CI with this error `No match for argument: jq
Error: Unable to find a match: jq`. Seems like it was not available in RH UBI images.